gradle/libs.versions.toml

```diff
@@ -1,11 +1,11 @@
 [versions]
-accompanist = "0.10.0.compose-7352378-SNAPSHOT"
+accompanist = "0.10.0"
 androidxhilt = "1.0.0"
-androidxlifecycle = "2.4.0-SNAPSHOT"
+androidxlifecycle = "2.4.0-alpha01"
 androidxtest = "1.3.0"
 coil = "1.2.1"
-compose = "1.0.0-SNAPSHOT"
-composesnapshot = "7352378" # a single character = no snapshot
+compose = "1.0.0-beta07"
+composesnapshot = "-" # a single character = no snapshot
```

Comment:
offtopic: why not empty?

Reply:
version catalogs doesn't allow empty strings 🤷